File: algorythm/Janken_algorythm/Janken_Structure.py
#! /usr/bin/python3

import time
import random

LOOP_COUNT = 1000


start = time.time()

for _ in range(LOOP_COUNT):

    user = random.randint(1,4)
    computer = random.randint(1,4)

    gu = 1
    tyoki = 2
    pa = 3

    draw = 1
    win = 2
    lose = 3

    judge = ([ draw, win, lose],[ lose, draw, win],[win, lose, draw])

    if (judge[user - 1][computer - 1] == draw):
        print("あいこ\n")
    elif (judge[user - 1][computer - 1] == win):
        print("勝ち\n")
    elif (judge[user - 1][computer - 1] == lose):
        print("負け\n")

end = time.time()
print("経過時間:", (end - start))

File: algorythm/Janken_algorythm/Janken_Structure.java
public class Janken_Structure {
    public static void main(String...args) {
        int gu = 1;
        int tyoki = 2;
        int pa = 3;

        int draw = 1;
        int win = 2;
        int lose = 3;

        int user = new java.util.Random.randint(3);
        int computer = new java.util.Random.randint(3);

        int judge[][] =  new int [3][3];

            judge[0][0] = draw;
            judge[0][1] = win;
            judge[0][2] = lose;

            judge[1][0] = lose;
            judge[1][1] = draw;
            judge[1][2] = win;

            judge[2][0] = win;
            judge[2][1] = lose;
            judge[2][2] = draw;



    }


}
